INSTALL_MACOS.md

```diff
@@ -330,8 +335,9 @@ curl -k -L https://github.com/NatronGitHub/OpenColorIO-Configs/archive/Natron-v2
 mv OpenColorIO-Configs-Natron-v2.5 OpenColorIO-Configs
 ```
 
+## Building with qmake (for Qt4 and Qt5)
```

Comment:
QMake is not supported for Qt5, I think it's better to remove its mention so it doesn't confuse people.

Reply:
qmake is still supported on Qt5, and is supported for building apps on Qt6
Refs:
- Qt5 https://wiki.qt.io/Qt5_Build_System
- Qt6 https://doc.qt.io/qt-6/cmake-manual.html

Also note that this is currently the only way to configure/build on macOS.